### PR TITLE
🔧 switch to tailwind & fix histoire darkmode

### PR DESCRIPTION
**Pull request by roiLeo**
- ref #8222
- migrate neoSwtich to use tailwind class
- fix darkmode toggle in histoire preview
- fix darkmode vars
- remove unecessary css class

Screenshot 2023-12-02 at 11-43-06 NeoSwitch › NeoSwitch Histoire
Screenshot 2023-12-02 at 11-42-59 NeoSwitch › NeoSwitch Histoire

Screenshot 2023-12-02 at 11-53-48 Create carbonless NFTs
Screenshot 2023-12-02 at 11-54-09 Create carbonless NFTs

### CODE REVIEW

**Comment by roiLeo** [comment]
@shashkovdanil check please.

**Review by exezbcz — APPROVED** [review]

File: libs/ui/src/scss/tailwind.scss [hunk 1]
```diff
@@ -58,54 +58,11 @@
     --toggle-active-switch: #d9d9d9;
   }
 
-  @media (prefers-color-scheme: dark) {
```
[the remark this answers is not in the document]
Comment (roiLeo) [reply libs/ui/src/scss/tailwind.scss:61]: why did you want that in first place? `nuxt-color` define html class theme from `prefers-color-scheme` by default

**Comment by exezbcz** [comment]
noticed one small thing 

image

- the outer stroke is not true black - its a shade of grey

**Comment by exezbcz** [comment]
@roiLeo 
image


ignore the size, the size we are using currently is the correct one 

https://www.figma.com/file/JXNtryaNYhY0XJNArzvPQx/Handoff?type=design&node-id=1287-18959&mode=design&t=gzdTNFe3BzKrTp23-11

**Comment by roiLeo** [comment]
hmm.. something that I don't understand
Capture d’écran 2023-12-02 à 9 49 20 PM

**Comment by exezbcz** [comment]
@roiLeo okay once again :D 


the component in the histoire preview has different color of the stroke 

image

**Comment by roiLeo** [comment]
> @roiLeo okay once again :D 
> 
> 
> the component in the histoire preview has different color of the stroke 
> 
> image
> 

Maybe a screenshots issue? check any Switch in app to see if there IS a different outter strike

**Pull request merged**